src/routes/PersonalPages/index.tsx

```diff
@@ -176,6 +192,16 @@ const Competences = ({ routes }: RoutesNavigationListProps) => {
         simul accusata no ius. Volumus corpora per te, pri lucilius salutatus iracundia ut. Mutat posse voluptua quo cu,
         in albucius nominavi principes eum, quem facilisi cotidieque mel no.
       </p>
+      <h2 className="mb-6 text-heading-3">Osaamiseni lähteiden mukaan</h2>
+      <Accordion title={<span className="truncate text-heading-5 text-secondary-gray">Työpaikkojen mukaan</span>}>
+        <span className="mb-5 mt-3 flex border border-b-2 border-inactive-gray"></span>
+        <div className="flex flex-wrap gap-4">
+          {hardcodedCompetenceTags.map((val, index) => {
+            return <Tag label={val} key={index} variant="added" onClick={() => console.log('clicked:', val)} />;
```

Comment:
Does it matter which we use as a key? Value or index, as both should be unique here.